cmd/csi-node-driver-registrar/main.go

```diff
@@ -53,6 +54,7 @@ var (
 	healthzPort             = flag.Int("health-port", 0, "(deprecated) TCP port for healthz requests. Set to 0 to disable the healthz server. Only one of `--health-port` and `--http-endpoint` can be set.")
 	httpEndpoint            = flag.String("http-endpoint", "", "The TCP network address where the HTTP server for diagnostics, including the health check indicating whether the registration socket exists, will listen (example: `:8080`). The default is empty string, which means the server is disabled. Only one of `--health-port` and `--http-endpoint` can be set.")
 	showVersion             = flag.Bool("version", false, "Show version.")
+	logFormatJSON           = flag.Bool("log-format-json", false, "set log formatter to json")
```

Comment:
It looks like component base already supports a log format flag, so I'm not sure we need to explicitly add one here: 

https://github.com/kubernetes/component-base/blob/master/logs/options.go#L89

I think it would be better if we switch to component-base log pkg instead of using klog directly.

Reply:
sure, let me do that.

Reply:
Updated, PTAL!